account_move_journal_template/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Automatically create Journal entries and items when posting out-invoices.
```

Comment:
This always happens when posting invoices. make somehow clear that it are extra moves dependent on the products in the invoice lines.